features/facebookImport/src/views/ministories/activityLocations.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import BasicDataCountTable from "../../components/basicDataCountTable/basicDataCountTable.jsx";
+import analysisKeys from "../../model/analyses/utils/analysisKeys";
+import { SingleDataStory } from "./singleDataStory.jsx";
+
+class ActivityLocationsMinistory extends SingleDataStory {
+    constructor(props) {
+        super(props, analysisKeys.locationsData);
+    }
+    get title() {
+        return "Session Activity Locations";
+    }
+    renderSummary() {
+        return (
+            <BasicDataCountTable
+                title={
+                    "Locations contained in session activities, like log-in or log-out."
```

Comment:
Again, no translation here?

Reply:
I think it's supposed to be a _tech demo_ -> Can we not mark it as such?